include/download.mk

```diff
@@ -214,14 +214,15 @@ endef
 
 # Only intends to be called as a submethod from other DownloadMethod
 define DownloadMethod/rawgit
-	echo "Checking out files from the git repository..."; \
+	gitdir="$(DL_DIR)/git/`basename $(URL)`"; \
```

Comment:
`basename $(URL)` is probably too short and it's going to cause namespace/repo clashes

Reply:
> `basename $(URL)` is probably too short and it's going to cause namespace/repo clashes

any suggestion? maybe $(FEED)-$(PACKAGE).git or $(FEED)-`basename $(URL)` ?